ci/pipeline

```diff
@@ -75,13 +75,14 @@ def uploadTarballToS3(): Option[awsClient.Artifact] = utils.stage("Upload Packag
 /**
  * Packages Marathon and uploads its artifacts alongside sha1 checksum to S3.
  *
+ * @param snapshot Indicates whether we want to upload the snapshot.
  * @return Version and artifact description of Marathon build.
  */
-def createAndUploadPackages(): (String, Option[awsClient.Artifact]) = {
+def createAndUploadPackages(snapshot: Boolean = true): (String, Option[awsClient.Artifact]) = {
   val version = createPackages()
   val docsPath = PACKAGE_DOCS_DIR / s"marathon-docs-$version.tgz"
-  val artifact = uploadTarballToS3()
-  awsClient.upload(docsPath)
+  val artifact = if(snapshot) uploadTarballToS3() else None
```

Comment:
shouldn't this condition be inverted? The title says we do NOT want to upload when snapshot build but this says if snapshot, then upload... or am I missing something?

Reply:
Good point. The logic is as I want it but the name `snapshot` is not fitting 🤔.